themes/default/content/blog/deprecating-non-project-mode/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Deprecating non-project mode from DIY backends"
+date: 2024-02-19
+meta_desc: Read about the plans for the removal of non-project mode.
+authors:
+    - fraser-waters
+tags:
+    - features
+    - pulumi-releases
+---
+
+Last year we released [project support for self-managed backends](/blog/project-scoped-stacks-in-self-managed-backend/), we now plan to remove support for non-project mode.
+
+<!--more-->
+
+Users using a DIY backend (previously referred to as self-managed) may have seen a warning in recent releases of Pulumi:
+
+```
+warning: Legacy non-project mode support for DIY backends are due to be deprecated this year.
+Please migrate to project mode by running 'pulumi state upgrade'.
+You can disable this warning by setting PULUMI_DIY_BACKEND_ACKNOWLEDGE_LEGACY_WARNING to true.
+```
+
+To assuage any concerns about this warning it is _specifically_ about running the DIY backend in non-project mode where all the state files are kept in one top level folder, we are _not_ removing support for DIY backends in general.
+
+As described in the warning we plan on deprecating and removing support for non-project mode this year.
+
+## Phase 1 - Warning
+
+The first phase of this deprecation has already been released. When opening a DIY state store if the system see it is still in non-project mode the above warning is printed. You can set the environment variable `PULUMI_DIY_BACKEND_ACKNOWLEDGE_LEGACY_WARNING` to silence this warning.
+
+## Phase 2 - Soft error
+
+In a few months we plan to make this a soft error. Opening a state store in non-project mode will result in an error like:
```

Comment:
```suggestion
In a few months we plan to make this a soft error. Opening a state store in non-project mode will result in an error like:
```

Would it make sense to make this a little bit more concrete, e.g. "In 2-3 months", so users have a more concrete idea of the changes that are coming?  A few months feels a little bit vague for users planning purposes.  Similar for the other steps below.